### PR TITLE
Retry on clone success but checkout failure

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: lib/git-fastclone.rb [hunk 1]
```diff
@@ -355,6 +355,7 @@ def with_git_mirror(url)
           error_strings = [
             'missing blob object',
             'remote did not send all necessary objects',
+            'packed object',
```
Comment (dnkoutso) [inline lib/git-fastclone.rb]: not _super_ happy on this message matching _but_ all of them start with `fatal:` so I am personally OK with that.

Comment (justinseanmartin) [reply lib/git-fastclone.rb]: What about `/packed object [a-z0-9]+ \(stored in .*?\) is corrupt/`?

Comment (dnkoutso) [reply lib/git-fastclone.rb]: test did not pass updating a bit to make it pass.

Comment (dnkoutso) [reply lib/git-fastclone.rb]: updated.

**Comment by justinseanmartin** [comment]
Version bump (either now or outside PR is fine)

**Comment by dnkoutso** [comment]
Bumped version.

**Pull request merged**